modules/tinymce/src/core/main/ts/fmt/ApplyFormat.ts

```diff
@@ -39,6 +40,22 @@ const canFormatBR = (editor: Editor, format: ApplyFormat, node: HTMLBRElement, p
   }
 };
 
+const applyStyles = (dom: DOMUtils, elm: Element, format: ApplyFormat, vars: FormatVars | undefined) => {
+  each(format.styles, (value, name) => {
+    dom.setStyle(elm, name, FormatUtils.replaceVars(value, vars));
+  });
+
+  // Needed for the WebKit span spam bug
+  // TODO: Remove this once WebKit/Blink fixes this
```

Comment:
Do we have a reference for this bug? Moving the code is usually a good time to evaluate whether the workaround is still required :)

Reply:
I can't remember exactly. WebKit at least used to produce span elements all over out of thin air. But looking at when that was introduced it was before we use filtering on copy/cut so it might not be a problem anymore we might be able to remove this.